index.bs

```diff
@@ -5457,11 +5572,52 @@ complete</dfn> steps given |navigable| and |navigation status|:
    </dd>
 </dl>
 
+<div algorithm>
+The [=remote end event trigger=] is the
+<dfn export>WebDriver BiDi download will begin</dfn> steps given |navigable| and
+|navigation status|:
+
+1. Let |navigation info| be the result of [=get the navigation info=] given
+   |navigable| and |navigation status|.
+
+1. Let |params| be a [=/map=] matching the
+   <code>browsingContext.DownloadWillBeginParams</code> production, with the
+   <code>context</code> field set to |navigation info|["<code>context</code>"], the
+   <code>navigation</code> field set to |navigation info|["<code>navigation</code>"],
+   the <code>timestamp</code> field set to
+   |navigation info|["<code>timestamp</code>"], the <code>url</code> field set to
+   |navigation info|["<code>url</code>"] and <code>suggestedFilename</code> field set
+   to |navigation status|'s <code>suggestedFilename</code>.
```

Comment:
I had some test updates planned for this at https://phabricator.services.mozilla.com/D264048 (before I knew your PR was touching this).

I think they should still be aligned with the proposed changes but Chrome is failing them in 2 spots:
- there is a navigation id set for the download created by a download attribute
- the navigation id for downloadWillBegin is different from the navigation id of navigationStarted for the navigation download. 

First item sounds ok, I guess you will update the implementation. But for the second item, do you agree the navigation ids in the 2 events should be equal?